examples/timer/__tests__/infinite_timer_game.test.js

```diff
@@ -5,15 +5,16 @@
 jest.useFakeTimers();
 
 it('schedules a 10-second timer after 1 second', () => {
+  jest.spyOn(global, 'setTimeout');
```

Comment:
The reason for this change is that the fake timer functions in current implementation are mock functions, a feature I've purposefully _not_ implemented in the Lolex version.